data/engines.txt

```diff
@@ -389,7 +389,7 @@ outfit "A370 Atomic Thruster"
 	"flare sprite" "effect/atomic flare/medium"
 		"frame rate" 12
 	"flare sound" "atomic medium"
-	description "This thruster is small enough to fit in a light warship or freighter, but powerful enough to move any but the largest of ships. Naturally, such performance comes at an exorbitant price."
+	description "This thruster is small enough to fit in a light warship or freighter, but powerful enough to move any ship in the sky, excluding only the very largest of capital ships. Naturally, such performance comes at an exorbitant price."
```

Comment:
I'm not a huge fan of it either, but I can't think of anything else right now. If someone else has a suggestion, I'm very open to changing it.